### ISSUE TITLE
Disable auto-close parentheses only works for text editor not notebook

### ISSUE THREAD

**Issue by aarchiba** [jupyterlab/jupyterlab#7548]
## Description

The current version of Jupyterlab (1.2.3) has a Settings menu option "Auto Close Brackets for Text Editor". But this does not affect auto closing brackets in the notebook, and no analogous option is present (nor is it obvious that this option doesn't do that - does "text editor" mean something different from the way I edit text in the notebook? Apparently.). To fix the notebook's behaviour one needs to look in the Advanced Settings editor, and one needs to know to look under Notebook rather than Text Editor, and one needs to know enough about JSON syntax to produce a correctly formatted settings file. Please add an analogous setting to turn off this behaviour in the Notebook.

**Comment by jasongrout** [comment]
> does "text editor" mean something different from the way I edit text in the notebook?

Yes, "text editor" refers to the editor for text files, i.e., the editor when you create a new markdown or python file, for example.

It sounds like a new menu entry for Notebook editor settings might be good, or alternatively a much friendlier interface for general advanced settings.

Thanks for bringing this up. We obviously can't put every setting into the Settings menu, and it is hard to know which settings are important/common enough to surface into that menu without user feedback.

Does anyone else wish this particular (or some other particular) notebook editor setting was surfaced in the Settings menu?
